partials/nav-global.php

```diff
@@ -49,7 +49,12 @@
 							<div id="header-search">
 								<form class="form-search" role="search" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>">
 									<div class="input-append">
-										<input type="text" placeholder="<?php _e('Search', 'largo'); ?>" class="input-medium appendedInputButton search-query" value="" name="s" /><button type="submit" class="search-submit btn"><?php _e('GO', 'largo'); ?></button>
+										<input type="text" placeholder="<?php _e('Search', 'largo'); ?>"
+										aria-label="Searchbox"
```

Comment:
Needs to be a translatable string. @joshdarby should this be something like "Sitewide Search"?

Reply:
Yea, I think it should be something more specific.